index.html

```diff
@@ -1,28 +1,197 @@
+        </button>
+        <ul class="nav-links d-non d-md-flex">
+          <li><a href="index.html" class="scroll">Home</a></li>
+          <li><a href="#speaker" class="scroll">Speaker</a></li>
+          <li><a href="about.html" class="scroll">about</a></li>
+        </ul>
+      </nav>
+    </div>
 
+    <div class="present" id="home"> <section class="bg w-100">
+      <p class=" slt">"Hello! Sharing world"</p>
+      <h1 class="top">Welcome to the best JavaScript BootCamp</h1>
+      <p class="info border-2">
+        Unlock your potential as a web developer with our comprehensive
+        JavaScript bootcamp. From novice to expert, we'll guide you through the
+        ins and outs of the language and give you the skills you need to create
+        dynamic and interactive web applications.
+      </p>
+      <h2>2023.04.01 (THU) ~ 05.20 (FRI)</h2></section>
+    </div>
 
+    <section class="course">
+      <h3 class="entete">Course Program</h3>
+      <hr />
+      <div class="program-container">
+        <!-- <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-solid fa-code"></i>
+            <h3 class="mb-0">Basic</h3>
+          </div>
+          <p class="mb-0">
+            This includes data types, variables, operators, control structures,
+            functions, and arrays.
+          </p>
+        </div> -->
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-brands fa-square-js"></i>
+            <h3 class="mb-0">DOM</h3>
+          </div>
+          <p class="mb-0">
+            This involves using JavaScript to modify the structure and content
+            of web pages, including adding, removing, and modifying HTML
+            elements.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-brands fa-react"></i>
+            <h3 class="mb-0">Framework</h3>
+          </div>
+          <p class="mb-0">
+            learning popular JavaScript frameworks and libraries such as React,
+            Vue.js, and Angular, which help development of web applications.
+          </p>
+        </div>
+        <div class="program d-flex flex-row flex-md-column">
+          <div
+            class="col-4 d-flex flex-column justify-content-center align-items-center"
+          >
+            <i class="fa-sharp fa-solid fa-server"></i>
+            <h3 class="mb-0">ServerSide</h3>
+          </div>
+          <p class="mb-0">
+            Using JavaScript to build server-side applications using Node.js,
+            which allows JavaScript tofull-stack web applications.
```

Comment:
- [ ] You can reduce the font size of the text in each card  and increase the padding around the elements so that the elements don't appear to be squeezed

![image](https://user-images.githubusercontent.com/98453979/225683902-81dc686a-e5e1-4226-8bb1-7cae9edb3cd4.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/225684084-caebaffe-0e11-48fa-817f-8381498563a0.png)

Reply:
Right